apps/pgsql/patroni/openshift/deployment.yaml

```diff
@@ -365,7 +293,7 @@ parameters:
   name: PVC_SIZE
   value: 1Gi
 - name: STORAGE_CLASS
-  value: gluster-block
+  value: netapp-file-standard
```

Comment:
Please update to `netapp-block-standard`

Reply:
done, and tested a new deployment by applying `deployment-prereq.yaml` and `deployment.yaml` in that order.